lib/easel_provider.dart

```diff
@@ -551,12 +551,12 @@ class EaselProvider extends ChangeNotifier {
 
   void onVideoThumbnailPicked() async {
     final result = await fileUtilsHelper.pickFile(NftFormat.supportedFormats[0]);
-    if (result != null) {
-      final loading = Loading().showLoading(message: kCompressingMessage);
-      final file = await fileUtilsHelper.compressAndGetFile(File(result.path!));
-      setVideoThumbnail(file);
-      loading.dismiss();
-    }
+
+    if (result == null) return;
```

Comment:
Show error message.

Reply:
Resolved as per now. will consider it after merging with other modules.